ui/src/components/ProvisionerDetailsCard/index.jsx

```diff
@@ -30,8 +30,8 @@ import { ACTION_CONTEXT } from '../../utils/constants';
     fontSize: '0.7rem',
   },
   headline: {
-    paddingLeft: theme.spacing.triple,
-    paddingRight: theme.spacing.triple,
+    paddingLeft: theme.spacing.double,
+    paddingRight: theme.spacing.double,
```

Comment:
This change is out of scope for this PR and I think it's already been handled by another PR. We can remove these style changes.